js_modules/dagster-ui/packages/ui-core/src/runs/LogsToolbar.tsx

```diff
@@ -75,34 +68,17 @@ export const LogsToolbar: React.FC<ILogsToolbarProps> = (props) => {
     children,
   } = props;
 
-  const [
-    initialComputeLogType,
-    setInitialComputeLogType,
-  ] = useStateWithStorage(INITIAL_COMPUTE_LOG_TYPE, (value: any) =>
-    typeof value === 'string' ? (value as LogType) : LogType.stderr,
-  );
-
-  const setComputeLogType = React.useCallback(
-    (logType: LogType) => {
-      setInitialComputeLogType(logType);
-      onSetLogType(logType);
-    },
-    [onSetLogType, setInitialComputeLogType],
-  );
-
-  const activeItems = React.useMemo(
-    () => new Set([logType === LogType.structured ? logType : initialComputeLogType]),
-    [logType, initialComputeLogType],
-  );
+  const activeItems = React.useMemo(() => new Set([logType]), [logType]);
 
   return (
     <OptionsContainer>
       <Box margin={{right: 12}}>
         <ButtonGroup
           activeItems={activeItems}
           buttons={[
-            {id: LogType.structured, icon: 'list', tooltip: 'Structured event logs'},
-            {id: initialComputeLogType, icon: 'wysiwyg', tooltip: 'Raw compute logs'},
+            {id: LogType.structured, icon: 'view_list', label: 'STRUCTURED'},
+            {id: LogType.stdout, icon: 'console', label: 'STDOUT'},
+            {id: LogType.stderr, icon: 'warning', label: 'STDERR'},
```

Comment:
May not have made it into the designs but Nick asked for a last minute change to make stderr + stdout lowercase instead of all caps. And we decided to use the word `Events` in place of `Structured`.

Reply:
Got it, I will update this